v_4_5_0/files/conf/k8s-addons

```diff
@@ -9,6 +9,19 @@ KUBECTL={{ .RegistryDomain }}/giantswarm/docker-kubectl:f5cae44c480bd797dc770dd5
 # wait for healthy master
 while [ "$(/usr/bin/docker run -e KUBECONFIG=${KUBECONFIG} --net=host --rm -v /etc/kubernetes:/etc/kubernetes $KUBECTL get cs | grep Healthy | wc -l)" -ne "3" ]; do sleep 1 && echo 'Waiting for healthy k8s'; done
 
+# label namespaces (required for network egress policies)
+NAMESPACES="kube-system giantswarm"
```

Comment:
Why not label `default` too? (in case customers can add netpol for that)

Reply:
I don't care about customer's needs :trollface: 
updating...